submissions/victorAienobe/phpDatabase/saveCookie.php

```diff
@@ -0,0 +1,40 @@
+<?php
+   
+    include_once('connection.php');
+    
+    //Querying the database to select all information available in the table
+    try {
+        $sql= "SELECT * FROM customer_details";
+        $result = $pdo->query($sql);
+        
+    // setting up the cookie to save the result gotten from the databse
+    setcookie("customer", json_encode($result), time()+7200, "/");
+    include_once('accessCookie.php');
+
+        if($result-> rowCount() > 0)  {
```

Comment:
Your implementation still doesn't satisfy this `Ensure that you are not hitting the database each time there is a page reload as long as the cookie is still set or hasn't expired yet` criteria. What is having currently, if I reload the page, the `$result` is still coming from the SQL query